### ISSUE TITLE
OrientDB SQLs for compatibility test

### ISSUE THREAD

**Issue by wuxianliang** [agershun/alasql#194]
Thank you for your helps!

**Comment by agershun** [comment]
@wuxianliang Do you know, are there any available OrientDB SQL tests?

**Comment by wuxianliang** [comment]
I have write a post in OrientDB Google Group. This issue is for community's helps. The post is about OrientDB SQL tests and need to be approved to appear.

**Comment by wuxianliang** [comment]
In my understanding, OrientDB SQL tests mean concrete OrientDB SQL statements. Right? @lvca @luigidellaquila

**Comment by agershun** [comment]
Yes. The real samples make the development easier, because you know will be in result

Отправлено с iPhone

**Comment by luigidellaquila** [comment]
Hi @wuxianliang 

I don't know exactly what kind of tests you are thinking about, if you need practical examples you can start from something like this

http://pizzaconnections.net/2015/03/27/148/

if you need real test cases, there are a bunch of them here ;-)

https://github.com/orientechnologies/orientdb/tree/master/core/src/test/java/com/orientechnologies/orient/core/sql

Luigi

**Comment by agershun** [comment]
@luigidellaquila Thank you very much, this is exactly what we need! 

I would be happy to talk to you sometime on the next week, when we will have first results.

**Comment by luigidellaquila** [comment]
@agershun I'll be happy to talk to you, let me know when you have something 

Luigi

**Comment by mathiasrw** [comment]
Did @luigidellaquila and @agershun ever had a chat?
